internal/error.go

```diff
@@ -606,48 +609,54 @@ func IsRetryable(err error, nonRetryableTypes []string) bool {
 		return false
```

Comment:
IsRetryable is used only by backoff.Retry function which in turn I used only by API calls to the service. I don't believe it should ever receive any of these errors.